pyproject.toml

```diff
@@ -0,0 +1,110 @@
+[tool.poetry]
+name = "pyrorisks"
+version = "0.1.0"
+description = "Data pre-processing pipelines and models for wildfire forecasting and monitoring"
+authors = ["Pyronear <contact@pyronear.org>"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [{include = "pyrorisks"}, {include = "app"}]
+classifiers=[
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+keywords=["data science", "time series", "machine learning"]
+
+
+[tool.poetry.dependencies]
+python = "^3.10"
+requests = "^2.31.0"
+geopandas = "1.0.1"
+boto3 = "^1.28.62"
+shapely = "^2.0.4"
+rasterio = [{path = "./build/rasterio-1.3.10-cp39-cp39-manylinux2014_x86_64.whl", platform = "linux", python = ">=3.10 <3.12"}]
+
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.4.2"
+black = "^23.9.1"
+pre-commit = "^3.4.0"
+mypy = "^1.10.0"
+ruff = "^0.5.1"
+pytest-cov = "^5.0.0"
+
+
+[tool.poetry.group.app.dependencies]
```

Comment:
[QUESTION] should we name it app or api?

Reply:
Both can work. I named it `app` to match the top-level directory and the fast API app/API convention, but `api` is straightforward!